DuckDuckGo/AutofillLoginSettingsListViewController.swift

```diff
@@ -349,6 +356,20 @@ final class AutofillLoginSettingsListViewController: UIViewController {
         }
     }
 
+    private func openSearchIfRequired() {
+        // Don't auto open search if user has selected an account
+        if selectedAccount != nil {
```

Comment:
Super minor suggestion - If you preferred, we could use a `guard` here, e.g:

```suggestion
        guard selectedAccount == nil else { return }
```